kata06/app/controllers/employees_controller.rb

```diff
@@ -0,0 +1,35 @@
+class EmployeesController < ApplicationController
```

Comment:
below this line, you can use the hook before_action like this:

```ruby
before_action :set_employe, only: i%[show edit update destroy]
```

Reply:
if you can explain me what that line means i will appreciated! (the "before_action" i already understand what that means)